appeals/web/src/server/appeals/appeal-details/appellant-case/site-area/site-area.validators.js

```diff
@@ -1,12 +1,13 @@
 import { createValidator } from '@pins/express';
 import { body } from 'express-validator';
+import { createTextInputValidator } from '#lib/validators/text-input-validator.js';
 
 export const validateSiteArea = createValidator(
-	body('siteArea')
-		.trim()
-		.notEmpty()
-		.withMessage('Provide the site area')
-		.bail()
-		.isNumeric()
-		.withMessage('Provide a number or decimal')
+	createTextInputValidator(
+		'siteArea',
+		'Enter the site area',
+		9,
```

Comment:
Should the character limit and/or error message come from a constant?

Reply:
Pulling char limit now from a constant. Error messages are consistently static within validators. Drew the refactoring line.